enginetest/enginetests.go

```diff
@@ -6492,7 +6492,19 @@ func runQueryPreparedWithCtx(
 	require := require.New(t)
 	parsed, err := parse.Parse(ctx, q)
 
-	if _, ok := parsed.(sql.Databaser); ok {
+	_, isInsert := parsed.(*plan.InsertInto)
+	_, isDatabaser := parsed.(sql.Databaser)
+	if bl, ok := parsed.(*plan.Block); ok {
```

Comment:
When is an insert ever inside a block node? What's going on here?

Reply:
this one also confuses me, daylon added Block last week for foreign keys

Reply:
looks like `*sqlparser.MultiAlterDDL` is now a thing, Block wraps arbitrarily nested statements?